common/tensorrt_common/include/tensorrt_common/tensorrt_common.hpp

```diff
@@ -92,18 +123,56 @@ using TrtUniquePtr = std::unique_ptr<T, InferDeleter<T>>;
 
 using BatchConfig = std::array<int32_t, 3>;
 
+/**
+ * @class TrtCommon
+ * @brief TensorRT common library
+ */
 class TrtCommon  // NOLINT
 {
 public:
+  /**
+   * @brief Construct TrtCommon.
+   * @param[in] mode_path ONNX model_path
+   * @param[in] precision precision for inference
+   * @param[in] calibrator pointer for any type of INT8 calibrator
+   * @param[in] batch_config configuration for batched execution
+   * @param[in] max_workspace_size maximum workspace for building TensorRT engine
+   * @param[in] buildConfig configuration including precision, calibration method, dla, remaining
+   * fp16 for first layer,  remaining fp16 for last layer and profiler for builder
+   * @param[in] plugin_paths path for custom plugin
+   */
   TrtCommon(
     const std::string & model_path, const std::string & precision,
-    std::unique_ptr<nvinfer1::IInt8EntropyCalibrator2> calibrator = nullptr,
+    std::unique_ptr<nvinfer1::IInt8Calibrator> calibrator = nullptr,
     const BatchConfig & batch_config = {1, 1, 1}, const size_t max_workspace_size = (16 << 20),
+    const BuildConfig & buildConfig = BuildConfig(),
     const std::vector<std::string> & plugin_paths = {});
 
+  /**
+   * @brief Load TensorRT engine
+   * @param[in] engine_file_path path for a engine file
+   * @return flag for whether loading are succedded or failed
```

Comment:
```suggestion
   * @return flag for whether loading are succeeded or failed
```

Reply:
@yukkysaito 
Thank you for pointing this out. I correct the spelling at 3f059adf7a831b959038a6380497ffbb1feea5e9